src/testExplorer.html

```diff
@@ -0,0 +1,15 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="height=device-height, width=device-width, initial-scale=1.0" />
+    <title>Azure Cosmos DB</title>
+    <link rel="shortcut icon" href="images/CosmosDB_rgb_ui_lighttheme.ico" type="image/x-icon" />
+  </head>
+
+  <body>
+    <switch-directory-pane params="{data: switchDirectoryPane}"></switch-directory-pane>
+
+    <iframe id="explorerMenu" name="explorer" class="iframe" title="explorer" src="explorer.html?v=1.0.1&platform=Test">
```

Comment:
I see now you are using `platform=Test`. Is there a reason for needing a new one? I think we should be using platform=Portal so the underlying Explorer code knows what to do. We have a lot of checks for == Portal for behavior